HHS/OPRE-OPS
# feat: ✨ adds confirmation modals, alerts, and notes to Approve Agreement page

## What changed

- alerts and confirmation modals for Cancel, Decline, and Approve buttons
- adds notes to alerts
- adds submitter's notes

## Issue

#2349 

## How to test

1. Goto Agreement 1 or 9
2. make a status or budget change
3. send to approval
4. Goto Review list and click on review card to see approval page
5. Approve/Decline/Cancel buttons actions will trigger a confirmation modal and Alert passing along the notes and change requests

## Screenshots

https://github.com/user-attachments/assets/f851337e-e43c-456b-a981-b8d8e1ff0fa1

## Definition of Done Checklist
